test/std/process/exec_test.ds

```diff
@@ -1,6 +1,11 @@
+if is_windows
+    command = set "cmd /c"
+else
+    command = set "sh -c"
+end
 
 fn test_echo_with_output
-    output = exec echo hello world
+    output = exec %{command} "echo hello world"
```

Comment:
asked before in the rust tests, i'm asking again in the duckscript tests.
do we really need the extra "?